recipe/meta.yaml

```diff
@@ -49,7 +48,8 @@ test:
     - alembic.util
   requires:
     - pip
-    - python <3.10
+    - python
+    - pytest
```

Comment:
Why are we adding `pytest` as a test requirement? Nothing in the test commands suggest that it's needed.

Reply:
`pytest` is marked as a test dependency in the upstream: 
https://github.com/sqlalchemy/alembic/blob/rel_1_8_1/setup.cfg#L116

Reply:
What happens when you build it without `pytest`? Do one of the imports require it? Or can you add the `pytest` command to the test commands?